### PR TITLE
Docs:  adding a section about disable rules for some files

### PR DESCRIPTION
**Pull request by welll**
**What is the purpose of this pull request? (put an "X" next to item)**

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:
 
**What changes did you make? (Give an overview)**
I have added a new block explaining that is possible disable some rules only for certain files.

**Is there anything you'd like reviewers to focus on?**
- Verify if the example, phrases, sentences; follow the current writing style.
- Verify if this section is not already available in another page, maybe I missed.
- Tell me if I should write a real world use-case:

```js
describe('Foo', () => {
  it('bar', () => {
    var foo = fooBar()
    expect(foo).to.be.true
  })
```

### CODE REVIEW

File: docs/user-guide/configuring.md [hunk 1]
```diff
@@ -448,6 +448,23 @@ foo(); /* eslint-disable-line example/rule-name */
 
 **Note:** Comments that disable warnings for a portion of a file tell ESLint not to report rule violations for the disabled code. ESLint still parses the entire file, however, so disabled code still needs to be syntactically valid JavaScript.
 
+
+### Disabling Rules Only for a Group of Files
+
+To disable rules inside of a configuration file for a group of files, use the `overrides` key along with a `files` key. For example:
+
+```json
+ "rules": [...],
```
Comment (platinumazure) [inline docs/user-guide/configuring.md]: The "rules" key is generally an object, not an array.

File: docs/user-guide/configuring.md [hunk 2]
```diff
@@ -448,6 +448,23 @@ foo(); /* eslint-disable-line example/rule-name */
 
 **Note:** Comments that disable warnings for a portion of a file tell ESLint not to report rule violations for the disabled code. ESLint still parses the entire file, however, so disabled code still needs to be syntactically valid JavaScript.
 
+
+### Disabling Rules Only for a Group of Files
+
+To disable rules inside of a configuration file for a group of files, use the `overrides` key along with a `files` key. For example:
+
+```json
```
Comment (platinumazure) [inline docs/user-guide/configuring.md]: I think it would be good to surround the entire JSON blob with curly braces.

**Comment by welll** [comment]
Hi @platinumazure , I am doing now what you have suggested. Thanks for reviewing

File: docs/user-guide/configuring.md [hunk 3]
```diff
@@ -448,6 +448,25 @@ foo(); /* eslint-disable-line example/rule-name */
...
+{
+ "rules": {...},
```
Comment (platinumazure) [inline docs/user-guide/configuring.md]: Apologies, but could this be indented one more space for consistency?

File: docs/user-guide/configuring.md [hunk 4]
```diff
@@ -448,6 +448,25 @@ foo(); /* eslint-disable-line example/rule-name */
...
+ "overrides": [
```
Comment (platinumazure) [inline docs/user-guide/configuring.md]: Apologies, but could this be indented one more space for consistency?

**Comment by welll** [comment]
@platinumazure , I have done your suggestions.

**Comment by Aladdin-ADD** [comment]
thanks for the contributing!

**Pull request merged**